src/transaction/components/TransactionProvider.tsx

```diff
@@ -77,7 +79,9 @@ export function TransactionProvider({
     : TRANSACTION_TYPE_CONTRACTS;
 
   // Retrieve wallet capabilities
-  const { walletCapabilities } = useOnchainKit();
+  const walletCapabilities = useCapabilitiesSafe({
+    chainId: chainId || baseSepolia.id,
```

Comment:
For follow up PR, we need to make sure we handle this correctly where:
- we prioritize the `chainId` from the Transaction component
- if that not present we use the `chainId` from `useOnchainKit()`
- which if not present at OnchainKit config defaults to BaseSepolia.

We can not cut a release if this is not present! Just FYI
cc @0xAlec @alessey @cpcramer @abcrane123